### PR TITLE
Major UI Update

### PR DESCRIPTION
**Pull request by jvsluis**
Major UI Overhaul. Includes:
- New layout for multistage forms
- New nav bar
- New job listing view
- Working filters
- Many more graphical bug fixes
- Improved general usability of app

### CODE REVIEW

**Review by DanielVette — APPROVED** [review]
awesome

File: app/src/app.js [hunk 1]
```diff
@@ -29,45 +38,26 @@ function Layout() {
...
-    <div>
-      <nav className="flex justify-between w-full p-1 bg-transparent px-4 py-2 text-xl font-bold">
-        <button onClick={() => navigate("/")}>
-          <h2>PrintLink3D</h2>
-        </button>
+    <>
+      <nav className="flex justify-between items-center w-full px-4 py-3">
+        <Link to="/">
+          <h2 className="text-4xl">
+            <span className="fg-brand-blue">Print</span><span className="fg-brand-purple">Link3D</span>
```
Comment (Jasper-Nelligan) [inline app/src/app.js:46]: Should be with a lower case 'd'

File: app/src/components/multistepform.js [hunk 2]
```diff
@@ -42,101 +42,67 @@ export default function MultiStepForm({
     if (Children.count(children) == 3) {
       console.log(step);
```
Comment (Jasper-Nelligan) [inline app/src/components/multistepform.js:43]: Console log here

File: app/src/components/jobCardList.js [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
   return (
-    <div className="mx-auto">
-      <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-4">
-        {jobs.map((job) => (
-          <JobCard
+    <div className=''>
+      <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-2 gap-4">
+      {
+        jobs.map((job, i) => {
+        if (filters["materials"].includes(job.material)) {
```
Comment (Jasper-Nelligan) [inline app/src/components/jobCardList.js:11]: We should be filtering for color as well

File: app/src/routes/browse.js [hunk 4]
```diff
@@ -42,12 +48,76 @@ export default function Browse() {
...
+
+      </MultiStepFormPage>
+
+      </MultiStepForm>
+    </div>
+  );
+}
+
+
+/*
```
Comment (Jasper-Nelligan) [inline app/src/routes/browse.js:117]: Commented out code here

File: app/src/routes/browse.js [hunk 5]
```diff
@@ -3,10 +3,16 @@ import React, { useState, useEffect } from "react";
...
 export default function Browse() {
   const [selectedJob, setSelectedJob] = useState(null);
   const [jobs, setJobs] = useState([]);
+  const [filters, setFilters] = useState({
+    "materials": ["PLA", "ABS", "PETG"],
+    "bid_order": 0,
+  });
 
   useEffect(() => {
     const unsubscribe = onSnapshot(
```
Comment (Jasper-Nelligan) [inline app/src/routes/browse.js:18]: Not sure what happened here but jobs no longer render on the board. Even after creating a new issue and refreshing the page, nothing shows up

**Pull request merged**